ecr/README.md

```diff
@@ -15,9 +14,10 @@ module "ecr" {
   # Optional
   mutable = false
   valid_days = 3
+  protected_tags = ["latest-master", "latest-production"] # will keep at least 1 of this tag
```

Comment:
nit pick

```suggestion
  protected_tags = ["latest-main", "latest-production"] # will keep at least 1 of this tag
```

Reply:
do we need to document, that `mutable` **must** be `true` for protected tags to work?